docs/_partners/integration-quality.md

```diff
@@ -0,0 +1,126 @@
+* Keep your platform and Zapier integration parallel to maintain a consistent experience for mutual users.
+    * Object and field names referenced in the integration should parallel names in the platform. For example, don’t name your trigger “New Lead” if they are referred to as “Contacts” in your platform. This makes it hard for users to find the functionality they want.
+    * Provide all available input fields in the platform in the integration step, including custom fields. 
+    * Match the field types in the platform to the integration. For example, if a field is a static dropdown in the platform, implement it as the same type within the integration.
+
+
+## Bugs
+
+As users encounter bugs or think of new features they’d like within the integration, some reach out to the Zapier support team. Those requests are logged in Zapier’s issue tracker, which you can see from the Bug & Feature Requests page of the integration’s developer platform. Here are a couple things to note about bugs:
+
+1. _When the number of open bugs goes above zero, it doesn’t really matter how many bugs you have._ The important datapoint is how many of your users are affected overall and what percentage of the app’s overall monthly active users that impacts.
+2. _Your past history on closing bugs doesn’t influence your current health score._ Running track in college doesn’t mean a darn thing to your doctor thirty years later if you roll in with high cholesterol because of your penchant for McDonalds and KFC. Same with bugs and integration health.
+
+Don’t forget the affected user counts on issues underrepresent the actual totals, as only a small portion of all users facing an issue take the time to contact support. If a significant number of users have made the effort to contact support to say they are affected by a bug in the integration, it should raise flags.
+
+### What we have learned
+
+Open bugs are one of the best indicators of an integration’s health. If users are contacting Zapier’s support team to tell us about problems, we take that as a strong signal something is broken and needs to be fixed.
+
+Regardless of the size of your integration’s user base, keeping the percent of monthly active users affected by open bugs under double figures (and ideally at 0) is recommended for a healthy integration.
+
+### Best Practices
+
+* Instead of a “set-it-and-forget-it” mentality, allocate ongoing resources in your team's product roadmap for the maintenance of your Zapier integration to avoid surprise work or gaps in functionality.
+* Can’t keep up with maintenance? Hire a Zapier expert to help you fix on-off bugs or maintain your integration.
+* Make sure the right people on your team are getting issue notifications. You can add emails to receive these under the [“Notification Settings”](https://cdn.zappy.app/1b35f51a79c3caead06fedeb1fe0cffb.png) of the Bugs & Feature Requests tab in the integration’s developer platform.
```

Comment:
#fyi: TPS has a backlog ticket to remove the `Notification Settings` section as it's redundant now that issues go to all Admins and the Collaborator role exists: https://zapierorg.atlassian.net/browse/TPS-1985

Reply:
oOoh, great catch - TY! Removing it altogether.